$OpenBSD: patch-gstreamer_CMakeLists_txt,v 1.3 2014/11/04 06:55:04 zhuk Exp $
Use right variable to get whole include directories list, including dependencies.
--- gstreamer/CMakeLists.txt.orig	Sat Sep  6 20:13:59 2014
+++ gstreamer/CMakeLists.txt	Tue Nov  4 09:48:51 2014
@@ -15,12 +15,12 @@
 
 include_directories(
       ${CMAKE_CURRENT_BINARY_DIR}
-      ${GSTREAMER_INCLUDE_DIR}
+      ${GSTREAMER_INCLUDE_DIRS}
       ${GSTREAMER_PLUGIN_VIDEO_INCLUDE_DIR}
       ${GSTREAMER_PLUGIN_AUDIO_INCLUDE_DIR}
       ${GSTREAMER_PLUGIN_PBUTILS_INCLUDE_DIR}
-      ${GLIB2_INCLUDE_DIR}
-      ${LIBXML2_INCLUDE_DIR}
+      ${GLIB2_INCLUDE_DIRS}
+      ${LIBXML2_INCLUDE_DIRS}
       ${X11_X11_INCLUDE_PATH})
 
 if(${PHONON_VERSION} VERSION_GREATER "4.6.49") # Pre-release 4.7 starts at 4.6.50
